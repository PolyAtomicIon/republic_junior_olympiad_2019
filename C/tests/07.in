18 1
104 298 743 224 29 735 70 739 239 858 806 964 376 107 804 841 598 265 