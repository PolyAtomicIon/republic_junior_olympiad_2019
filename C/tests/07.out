104 402 1145 1369 1398 2133 2203 2942 3181 4039 4845 5809 6185 6292 7096 7937 8535 8800 